src/rust/ide/src/view/integration.rs

```diff
@@ -867,6 +869,13 @@ impl Model {
         Ok(())
     }
 
+    fn documentation_selected_in_ui(&self, is_selected:&bool) -> FallibleResult {
+        debug!(self.logger, "Documentation selected.");
+        let graph_frp = &self.view.graph().frp;
+        graph_frp.set_navigator_disabled(is_selected);
+        Ok(())
+    }
+
```

Comment:
I think this is wrong place: this file is integration between controllers and view, but here we integrate two views: documentation and graph.

The proper place is src/rust/ide/view/src/project.rs: the view containing both documentation panel and graph.